starter-core/src/main/java/io/micronaut/starter/feature/config/Toml.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.config;
+
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.feature.FeaturePhase;
+import io.micronaut.starter.template.Template;
+import io.micronaut.starter.template.TomlTemplate;
+import jakarta.inject.Singleton;
+
+import java.util.function.Function;
+
+@Singleton
+public class Toml implements ConfigurationFeature {
```

Comment:
I think you have to override `getCategory()`

Reply:
looks like `ConfigurationFeature` already overrides it, with `Category.CONFIGURATION`